pkgs/applications/video/pitivi/default.nix

```diff
@@ -40,11 +40,11 @@ in stdenv.mkDerivation rec {
     python pygobject3 gst-python pyxdg numpy pycairo sqlite3 matplotlib
   ]);
 
-  preFixup = ''
-    wrapProgram "$out/bin/pitivi" \
-      --set GDK_PIXBUF_MODULE_FILE "$GDK_PIXBUF_MODULE_FILE" \
-      --prefix GI_TYPELIB_PATH : "$GI_TYPELIB_PATH" \
-      --prefix GST_PLUGIN_SYSTEM_PATH_1_0 : "$GST_PLUGIN_SYSTEM_PATH_1_0" \
-      --prefix XDG_DATA_DIRS : "$XDG_ICON_DIRS:$out/share:$GSETTINGS_SCHEMAS_PATH"
-  '';
+#  preFixup = ''
```

Comment:
This just means that this package builds but still has issues or hasn't been tested yet, so this just for reminding me. When I'm finished all commented code will be removed.